river/Config.zig

```diff
@@ -178,15 +178,22 @@ pub fn deinit(self: *Self) void {
     self.xkb_context.unref();
 }
 
+const FilterKind = enum {
```

Comment:
This enum combines two things that shouldn't be: The effect of the filter (floating, csd) and the string it is matched against (app-id, title). I think it would be a bit cleaner to keep that separate.